tests/subsys/mgmt/smp_reassembly/src/main.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <sys/byteorder.h>
+#include <net/buf.h>
+#include <mgmt/mcumgr/smp.h>
+#include <mgmt/mcumgr/buf.h>
+#include <mgmt/mgmt.h>
+#include "../../../../../subsys/mgmt/mcumgr/smp_reassembly.h"
+
+static struct zephyr_smp_transport zst;
+static uint8_t buff[CONFIG_MCUMGR_BUF_SIZE];
+#define TEST_FRAME_SIZE 256
+
+static struct net_buf *backup;
+
+/* The function is called by zephyr_smp_reassembly_complete to pass completed packet
+ * for further processing; since there is nothing to process the frame and the test
```

Comment:
..since there is nothing to process, (comma) the frame and the test...

Reply:
Fixed